### ISSUE TITLE
can you update to tensorflow v1.x

### ISSUE THREAD

**Issue by mxmxlwlw** [da03/Attention-OCR#31]
Hi, 
     can you update to tensorflow v1.x. That will help a lot.

**Comment by jvpoulos** [comment]
Check out https://github.com/SivanKe/Attention-OCR/pull/1

Also fixes bugs, adds gradient clipping, etc.
